hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1944,24 +1944,19 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 
 	// Reconcile the Ignition server
 	if !controlplaneOperatorManagesIgnitionServer {
-		releaseInfo, err := r.lookupReleaseImage(ctx, hcluster, releaseProvider)
-		if err != nil {
-			return ctrl.Result{}, fmt.Errorf("failed to lookup release image: %w", err)
-		}
 		if _, exists := hcp.Annotations[hyperv1.DisableIgnitionServerAnnotation]; !exists {
```

Comment:
fwiw, this check is part of the igniton server component predicate, we could drop it here.